test/src/test_validate_generated_sq.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import os
+import json
+import jsonschema
+
+schema_file = "json-schema/StructuredQuery-schema-v3.json"
+generated_sqs_dir = "target/resources/generated-sqs/"
+
+@pytest.fixture
+def schema():
+    loaded_schema = None
+    with open(schema_file, 'r') as file:
+        loaded_schema = json.load(file)
+    return loaded_schema
+
+def test_validate_sqs(schema):
+    sqs = []
+    for file in os.listdir(generated_sqs_dir):
```

Comment:
Now I don't see that the test fails if there are no SQ's. Is it possible to have a parameterized test like in Java so that we have one test for each SQ? If so one could see that the number of tests will be zero if no SQ's are found.